src/current/cockroachcloud/managed-backups.md

```diff
@@ -0,0 +1,382 @@
+---
+title: Managed Backups
+summary: Learn about CockroachDB Cloud managed backups.
+toc: true
+docs_area: manage
+---
+
+{% include cockroachcloud/backups/managed-backup-description.md %}
+
+You can configure the following in [{{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters](#standard-and-advanced-clusters):
+
+- The frequency of the backups to meet [recovery point objective (RPO)]({% link {{site.current_cloud_version}}/disaster-recovery-overview.md %}#resilience-strategy) requirements.
+- The retention of the backups to set how long Cockroach Labs retains the backups.
+
+In [{{ site.data.products.basic }} clusters](#basic-clusters), you can view the default daily managed backups in the Cloud Console.
+
+{{site.data.alerts.callout_info}}
+In addition to managed backups, you can take manual backups to your own storage bucket with self-managed backups. Refer to the [Take and Restore Self-Managed Backups]({% link cockroachcloud/take-and-restore-self-managed-backups.md %}) page.
+{{site.data.alerts.end}}
+
+## Managed backup settings
+
+You can modify the settings of managed backups for [{{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters](#standard-and-advanced-clusters).
+
+[{{ site.data.products.basic }} clusters](#basic-clusters) have a default configuration that you cannot modify.
+
+### Basic clusters
+
+Cockroach Labs will take a managed backup every 24 hours. By default, managed backups will be retained for 30 days in {{ site.data.products.basic }} clusters.
+
+Once a cluster is deleted, Cockroach Labs retains the backup for 30 days. If an organization is deleted, you will lose access to all of the managed backups that Cockroach Labs has taken of the cluster after 30 days.
```

Comment:
I'm not sure that it's losing access so much as we'll delete the backups/they will age out after 30d anyway due to the retention

Reply:
Is there any difference in cluster deletion vs. organization deletion regarding how CRL will retain backups? I only ask because the "lose access" language was a remnant from what we currently say... and I wondered why I had phrased like that.

Reply:
OK, as this is for Basic clusters, will just stick to "we retain for 30 days".